README.md

```diff
@@ -43,6 +43,8 @@ To configure your Ubiquibot to run this plugin, add the following to the `.ubiqu
     miscellaneous:
       maxConcurrentTasks: 3
       startRequiresWallet: true # default is true
+      botUsernames:
+        - ubiquibot[bot]
```

Comment:
What is it for?

Reply:
I'm best guessing that our bot name won't be used across all partner's installs so it's needed to allow them to filter their own bot name

![image](https://github.com/user-attachments/assets/124e5e69-02a1-4f1d-94c3-87b00537266d)

Reply:
I thought just use `.type === "Bot"` but felt this might be the preferred approach

Reply:
I think for now using `Bot` is okay because it is easy to make a typo or forget about this field. Having the two bots running is a case that should not be happening often, and actually both bots should able not to clash with themselves. I suggest using `Bot` for now and if needed we can start to add extra configuration parameters as you did here.

Reply:
I think its a lot more robust to use the ID not the name. Besides we have at least one planned name change coming up soon i.e. UbiquityOS Superintendent / Kernel

Reply:
So you'd rather I use a an ID approach or just remove this config option completely and we'll hardcode our bot info?

Reply:
To be honest I don't feel that I have sufficient context because I can't even see the README.md in the code changes view. What problem are you trying to solve?

Reply:
Basically it's very likely that partners have other github bots running not just ours, I was trying to restrict the scope so we parse only assignment events from our bot

Reply:
Then you should identify our bot only by its user ID number.